pkg/logcli/client/file.go

```diff
@@ -183,13 +183,18 @@ func (f *FileClient) GetOrgID() string {
 }
 
 type limiter struct {
-	n int
+	n       int
+	timeout time.Duration
```

Comment:
Q: I don't see it's getting set anywhere in the file client. Could lead to unexpected behavior when using Logcli with file client I think. Can you also fix it where `limiter` is getting instantiated? (and with some sane default if value is not provided?)

Reply:
good catch, ty. I modified it to always use the 5minutes timeout, wdyt?